utils/contrib-list.php

```diff
@@ -61,19 +61,24 @@ public function __invoke( $_, $assoc_args ) {
 		$contributors = array_merge( $contributors, $repo_contributors );
 
 		// Identify all command dependencies and their contributors
-		$response = Utils\http_request( 'GET', 'https://raw.githubusercontent.com/wp-cli/wp-cli/master/composer.json' );
+		$milestones = self::get_project_milestones( 'wp-cli/wp-cli', array( 'state' => 'closed' ) );
+		// Cheap way to get the latest closed milestone
+		$milestone = array_shift( $milestones );
+		$composer_lock_url = sprintf( 'https://raw.githubusercontent.com/wp-cli/wp-cli/v%s/composer.lock', $milestone->title );
+		WP_CLI::log( 'Fetching ' . $composer_lock_url );
+		$response = Utils\http_request( 'GET', $composer_lock_url );
 		if ( 200 !== $response->status_code ) {
 			WP_CLI::error( sprintf( 'Could not fetch composer.json (HTTP code %d)', $response->status_code ) );
 		}
 		$composer_json = json_decode( $response->body, true );
-		foreach( $composer_json['require'] as $package => $version_constraint ) {
-			if ( ! preg_match( '#^wp-cli/.+-command$#', $package ) ) {
+		foreach( $composer_json['packages'] as $_ => $package ) {
```

Comment:
Why assign the key to `$_` here? You could just as well omit the key assignment, unless I'm missing something...